README.md

```diff
@@ -152,12 +154,300 @@ Sample output:
+var url = 'iteles/followers' // a random username (of someone you should follow!)
+gs(url, function(err, data) {
+  console.log(data); // or what ever you want to do with the data
+})
+```
+
+Sample output:
+
+```js
+{ entries:
+   [ 'tunnckoCore', 'OguzhanE', 'minaorangina', 'Jasonspd', 'muntasirsyed', 'fmoliveira', 'nofootnotes',
+    'SimonLab', 'Danwhy', 'kbocz', 'cusspvz', 'RabeaGleissner', 'beejhuff', 'heron2014', 'joshpitzalis',
+    'rub1e', 'nikhilaravi', 'msmichellegar', 'anthonybrown', 'miglen', 'shterev', 'NataliaLKB',
+    'ricardofbarros', 'boymanjor', 'asimjaved', 'amilvasishtha', 'Subhan786', 'Neats29', 'lottie-em',
+    'rorysedgwick', 'izaakrogan', 'oluoluoxenfree', 'markwilliamfirth', 'bmordan', 'nodeco', 'besarthoxhaj',
+    'FilWisher', 'maryams', 'sofer', 'joaquimserafim', 'vs4vijay', 'intool', 'edwardcodes', 'hyprstack',
+    'nelsonic' ],
+  url: 'https://github.com/iteles/followers' }
+ok 1 iteles/followers count: 45
+```
+
+If the person has ***more than 51 followers*** they will have multiple pages of followers.
+The data will have a **next_page** key with a value such as:
+[/nelsonic/followers?**page=2**](https://github.com/nelsonic/followers?page=2)
+If you want to keep fetching these subsequent pages of followers,
+simply keep running the scraper:
+e.g:
+
+```js
+var url = 'alanshaw/followers' // a random username (of someone you should follow!)
+gs(url, function(err, data) {
+  console.log(data); // or what ever you want to do with the data
+  if(data.next_page) {
+    gs(data.next_page, function(err2, data2)) {
+      console.log(data2); // etc.
+    }
+  }
+})
+```
+
+### **Following**
+Want to know the list of people this person is `following` that's *easy* too!
+The url format is: `https://github.com/{username}/following`
+e.g: [https://github.com/iteles/**following**](https://github.com/iteles/following) or
+[https://github.com/nelsonic/following?**page=2**](https://github.com/nelsonic/following?page=2)
+(_where the person is following more than 51 people_ ...)
+
+Usage format is *identical* to `followers` (above) so here's an example
+of fetching page 3 of the results:
+
+```js
+var gs = require('github-scraper'); // require the module
+var url = 'nelsonic/following?page=3' // a random dude
+gs(url, function(err, data) {
+  console.log(data); // or what ever you want to do with the data
+})
+```
+
+Sample output:
+
+```js
+{
+  entries:
+   [ 'kytwb', 'dexda', 'arrival', 'jinnjuice', 'slattery', 'unixarcade', 'a-c-m', 'krosti',
+   'simonmcmanus', 'jupiter', 'capaj', 'cowenld', 'FilWisher', 'tsop14', 'NataliaLKB',
+   'izaakrogan', 'lynnaloo', 'nvcexploder', 'cwaring', 'missinglink', 'alanshaw', 'olizilla',
+   'tancredi', 'Ericat', 'pgte' 'hyprstack', 'iteles' ],
+  url: 'https://github.com/nelsonic/following?page=3',
+  next_page: 'https://github.com/nelsonic/following?page=4'
+}
+```
+
+### Starred Repositories
+
+The list of projects a person has *starred* a fascinating source of insight.
```

Comment:
meh. happy to accept commits/PRs in the future if people want to get free points.